WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -777,9 +777,15 @@ extension DashboardViewController {
         if blazeCampaignHostingController != nil {
             removeBlazeCampaignView()
         }
+
+        guard let navController = navigationController else {
```

Comment:
Nit: You can simplify this check:

```suggestion
        guard let navigationController else {
```